portfolio/src/main/webapp/js/custom-script.js

```diff
@@ -304,6 +389,8 @@ async function drawTimeSeriesChart() {
     const xAxis = [];
     const yAxis = [];
 
+    // TODO(briafassler): move globaltemp.csv into csv folder and change file
+    // path
```

Comment:
TODO for Bria in your PR?

Reply:
I noticed that from @briafassler9  last PR globaltemp.csv is within the webapp directory with our html files, I didn't want to go in and change her code so I left that there for future reference.